main.py

```diff
@@ -0,0 +1,44 @@
+import ssl
```

Comment:
bad practise to import the whole module while only 1 function is used, should be:
from ssl import ssl.get_server_certificate